rai_test_utils/rai_test_utils/models/sklearn/sklearn_model_utils.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+from sklearn import svm
+from sklearn.compose import ColumnTransformer
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LogisticRegression
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import (FunctionTransformer, OneHotEncoder,
+                                   StandardScaler)
+
+
+def create_sklearn_random_forest_classifier(X, y):
+    rfc = RandomForestClassifier(n_estimators=10, max_depth=4,
+                                 random_state=777)
+    model = rfc.fit(X, y)
+    return model
+
+
+def create_sklearn_svm_classifier(X, y, probability=True):
+    clf = svm.SVC(gamma=0.001, C=100., probability=probability,
+                  random_state=777)
+    model = clf.fit(X, y)
+    return model
+
+
+def create_kneighbors_classifier(X, y):
+    knn = KNeighborsClassifier()
+    model = knn.fit(X, y)
+    return model
+
+
+def create_sklearn_logistic_regressor(X, y, pipeline=False):
+    lin = LogisticRegression(solver='liblinear', random_state=777)
+    if pipeline:
+        lin = Pipeline([('lin', lin)])
+    model = lin.fit(X, y)
+    return model
+
+
+def create_sklearn_random_forest_regressor(X, y):
+    rfc = RandomForestRegressor(n_estimators=10, max_depth=4,
+                                random_state=777)
+    model = rfc.fit(X, y)
+    return model
+
+
+def create_titanic_pipeline(X_train, y_train):
+    def conv(X):
+        if isinstance(X, pd.Series):
+            return X.values
+        return X
+
+    many_to_one_transformer = \
+        FunctionTransformer(lambda x: conv(x.sum(axis=1)).reshape(-1, 1))
+    many_to_many_transformer = \
+        FunctionTransformer(lambda x: np.hstack(
+            (conv(np.prod(x, axis=1)).reshape(-1, 1),
+                conv(np.prod(x, axis=1)**2).reshape(-1, 1))
+        ))
+    transformations = ColumnTransformer([
+        ("age_fare_1", Pipeline(steps=[
+            ('imputer', SimpleImputer(strategy='median')),
+            ('scaler', StandardScaler())
+        ]), ["age", "fare"]),
```

Comment:
nit: maybe move some of these strings to constants, like "age", "fare" and others that repeat

Reply:
will probably leave these strings as is.